.eleventy.js

```diff
@@ -19,16 +19,16 @@ const UserConfig = require("@11ty/eleventy/src/UserConfig");
  * @param {UserConfig} eleventyConfig
  */
 module.exports = function (eleventyConfig) {
-  // Copy the `images`, and `fonts` folders to the output
-  // CSS isn't copied over, that's done inline via the base template.
-  eleventyConfig.addPassthroughCopy("images");
-  eleventyConfig.addPassthroughCopy("fonts");
+  // Copy the `assets` (includes images, fonts) folders to the output
+  eleventyConfig.addPassthroughCopy("assets/fonts");
+  eleventyConfig.addPassthroughCopy("assets/images");
```

Comment:
Only add some folders to the passthrough.  Adding 'assets' alone doesn't trigger a rebuild when CSS files are modified.